app/src/views/Signup.vue

```diff
@@ -59,7 +63,8 @@ export default {
         confirmedPassword: ""
       },
       error: "",
-      success: ""
+      success: "",
+      rules: [v => /\S+@(nd|saintmarys|hcc-nd).edu/.test(v) || 'Email must be under nd.edu, saintmarys.edu, or hcc-nd.edu']
```

Comment:
You mind adding backend validation as well?

Reply:
added